src/connector.js

```diff
@@ -0,0 +1,769 @@
+	}
+}
+
+if ( !querySummaryTemplateHTML ) {
+	if ( lang === "fr" ) {
+		querySummaryTemplateHTML = 
+			`<h2 tabindex="-1" id="wb-land" role="status" class="h4">%[numberOfResults] résultats de recherche pour "%[query]"</h2>`;
+	}
+	else {
+		querySummaryTemplateHTML = 
+			`<h2 tabindex="-1" id="wb-land" role="status" class="h4">%[numberOfResults] search results for "%[query]"</h2>`;
+	}
+}
+
+if ( !didYouMeanTemplateHTML ) {
+	if ( lang === "fr" ) {
+		didYouMeanTemplateHTML = 
+			`<p tabindex="-1" class="h4 mrgn-lft-md did-you-mean">Rechercher plutôt <button class="btn-link p-0">%[correctedQuery]</button> ?</p>`;
+	}
+	else {
+		didYouMeanTemplateHTML = 
+			`<p tabindex="-1" class="h4 mrgn-lft-md did-you-mean">Did you mean <button class="btn-link p-0">%[correctedQuery]</button> ?</p>`;
+	}
+}
+
+if ( !noQuerySummaryTemplateHTML ) {
+	if ( lang === "fr" ) {
+		noQuerySummaryTemplateHTML = 
+			`<h2 tabindex="-1" id="wb-land" class="h4">%[numberOfResults] résultats de recherche</h2>`;
+	}
+	else {
+		noQuerySummaryTemplateHTML = 
+			`<h2 tabindex="-1" id="wb-land" class="h4">%[numberOfResults] search results</h2>`;
+	}
+}
+
+if ( !previousPageTemplateHTML ) {
+	if ( lang === "fr" ) {
+		previousPageTemplateHTML = 
+			`<button class="page-button previous-page-button">Précédente<span class="wb-inv">: Page précédente des résultats de recherche</span></ button>`;
+	}
+	else {
+		previousPageTemplateHTML = 
+			`<button class="page-button previous-page-button">Previous<span class="wb-inv">: Previous page of search results</span></ button>`;
+	}
+}
+
+if ( !pageTemplateHTML ) {
+	if ( lang === "fr" ) {
+		pageTemplateHTML = 
+			`<button class="page-button">%[page]<span class="wb-inv">: Page %[page] des résultats de recherche</span></ button>`;
+	}
+	else {
+		pageTemplateHTML = 
+			`<button class="page-button">%[page]<span class="wb-inv">: Page %[page] of search results</span></ button>`;
+	}
+}
+
+if ( !nextPageTemplateHTML ) {
+	if ( lang === "fr" ) {
+		nextPageTemplateHTML = 
+			`<button class="page-button next-page-button">Suivante<span class="wb-inv">: Page suivante des résultats de recherche</span></ button>`;
+	}
+	else {
+		nextPageTemplateHTML = 
+			`<button class="page-button next-page-button">Next<span class="wb-inv">: Next page of search results</span></ button>`;
+	}
+}
+
+if ( !pagerContainerTemplateHTML ) {
+	if ( lang === "fr" ) {
+		pagerContainerTemplateHTML = 
+			`<div class="text-center" >
+				<p class="wb-inv">Pagination des résultats de recherche</p>
+				<ul id="pager" class="pagination mrgn-bttm-0">
+				</ul>
+			</div>`;
+	}
+	else {
+		pagerContainerTemplateHTML = 
+			`<div class="text-center" >
+				<p class="wb-inv">Search results pages</p>
+				<ul id="pager" class="pagination mrgn-bttm-0">
+				</ul>
+			</div>`;
+	}
+}
+
+// auto-create results section if not present
+if ( !resultsSection ) {
+	resultsSection = document.createElement( "section" );
+	resultsSection.setAttribute( "class", "results" );
```

Comment:
FYI - the DOM do provide a nice way to set value on a class attribute to an element

```suggestion
	resultsSection.className = "results";
```

or you can use the array method instead
```suggestion
	resultsSection.classList.add( "results" );
```